tests/ui/meta-stats.rs

```diff
@@ -0,0 +1,7 @@
+// build-pass
```

Comment:
could you put this test in `tests/ui/stats` to join the one lonely test in there?